services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/design-kits.js

```diff
@@ -24,7 +37,18 @@ const DesignKits = ({ libraryData, navData }) => {
 
   const pageHeader = pageHeaders?.library ?? {}
 
-  const { name } = libraryData.content
+  const { name, designKits } = libraryData.content
+
+  const groupedDesignKits = groupBy(
+    Object.entries(designKits).map((key) => ({ ...key[1] })),
+    (kit) => kit.tool
+  )
```

Comment:
```suggestion
  const groupedDesignKits = groupBy(
    Object.values(designKits),
    (kit) => kit.tool
  )
```
or if you need it to be spread:
```
 const groupedDesignKits = groupBy(
    Object.values(designKits).map((val) => ({ ...val })),
    (kit) => kit.tool
  )
```